skeleton/src/main/java/uk/gov/hmcts/${{ values.product }}/${{ values.component | replace("-", "") }}/config/OpenAPIConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.${{ values.product}}.${{ values.component | replace("-", "") }}.config;
+
+import io.swagger.v3.oas.models.ExternalDocumentation;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.${{ values.product}}.${{ values.component | replace("-", "") }}.Application;
+
+@Configuration
+public class OpenAPIConfiguration {
+
+    @Bean
+    public OpenAPI openAPI() {
+        return new OpenAPI()
+            .info(new Info().title("rpe demo")
+                      .description("rpe demo")
+                      .version("v0.0.1")
+                      .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
+            .externalDocs(new ExternalDocumentation()
+                              .description("README")
+                              .url("https://github.com/hmcts/template-spring-boot"));
```

Comment:
these all need to be using variables that are replaced when generated.
(except license and version)